src/components/Maze/Maze.tsx

```diff
@@ -13,63 +13,74 @@ import './Maze.scss';
 
 const CameraController = () => {
   const { camera, gl } = useThree();
-  useEffect(
-    () => {
-      const controls = new OrbitControls(camera, gl.domElement);
-  
-      controls.minDistance = 5;
-      controls.maxDistance = 30;
-      controls.keyPanSpeed = 10;
-
-      //uncomment to limit orbit rotation
-      // controls.maxAzimuthAngle = (Math.PI / 360) * angle;
-      // controls.minAzimuthAngle = (Math.PI / 360) * -angle;
-
-      controls.maxPolarAngle = (Math.PI / 360) * 180 ;
-      
-      return () => {
-        controls.dispose();
-      };
-    },
-    [camera, gl]
-  );
+  useEffect(() => {
+    const controls = new OrbitControls(camera, gl.domElement);
+
+    controls.minDistance = 5;
+    controls.maxDistance = 30;
+    controls.keyPanSpeed = 10;
+
+    //uncomment to limit orbit rotation
+    // controls.maxAzimuthAngle = (Math.PI / 360) * angle;
+    // controls.minAzimuthAngle = (Math.PI / 360) * -angle;
+
+    controls.maxPolarAngle = (Math.PI / 360) * 180;
+
+    return () => {
+      controls.dispose();
+    };
+  }, [camera, gl]);
   return null;
```

Comment:
You might not have to return null here, generally the return statement in a `useEffect` function is a function that runs on the component's unmounting from the dom.  Shouldn't be an issue either way, just worth noting.

Reply:
Removing the return is giving `JSX element type 'void' is not a constructor function for JSX elements.` from <CameraController />